src/armature/Node.ts

```diff
@@ -355,18 +365,23 @@ export class Node {
                 )
             ),
 
-            vec3.fromValues(
-                Math.sqrt(
-                    Math.pow(position[0], 2) + Math.pow(position[1], 2) + Math.pow(position[2], 2)
-                ),
-                1,
-                1
-            )
+            vec3.fromValues(Math.max(1e-6, vec3.length(position)), 1, 1)
```

Comment:
Why did you impose a max on this?

Reply:
If you have a child node that is also at the root of the parent, then the bone connecting them has length 0 and can't be inverted. This is to make sure the bone doesn't entirely disappear into two dimensions.

Reply:
Ahh, sounds good to me, that makes sense. Could you maybe copy paste the above comment in the code so future viewers understand the decision too?